executor/src/main/java/org/k8loud/executor/service/MapperServiceImpl.java

```diff
@@ -1,13 +1,32 @@
 package org.k8loud.executor.service;
 
 import data.ExecutionRQ;
+import lombok.extern.slf4j.Slf4j;
 import org.k8loud.executor.action.Action;
+import org.k8loud.executor.action.ActionHelper;
 import org.springframework.stereotype.Service;
 
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
+
+@Slf4j
 @Service
 public class MapperServiceImpl implements MapperService {
+    private final ActionHelper actionHelper = new ActionHelper();
+
     @Override
     public Action map(ExecutionRQ executionRQ) {
-        return null;
+        String collectionName = executionRQ.getCollectionName();
```

Comment:
collectionName is e.g. `kubernetes`?

Reply:
Yes, each ruleset will have its collectionName to allow duplicate names.
For example `kubernetes.killAll`, `openstack.killAll` - both actions will terminate all managed processes so it's reasonable to give them the same names. Without collectionName there would be a naming clash.